### DISCUSSION TITLE
PhotoPrism not indexing all videos in a folder

### DISCUSSION THREAD

**Discussion by srahimeen** [photoprism/photoprism#4010] [Q&A]
#### 1. What is not working as documented?

I am running PhotoPrism on my Synology DS923+. PhotoPrism Build 231128-f48ff16ef.

I have PhotoPrism pointing to a folder called "Google Photos" on my NAS volume.

![image](https://github.com/photoprism/photoprism/assets/9031145/7ac50596-5214-4c39-b52a-6f1e8a0758fd)

Within this folder I have another folder "Google Photos/2023-2024 Melbourne Videography" which contains a lot of videos and a lot of them are >1GB. There are some json files as well, which is generated by Google during the Takeout process.

![image](https://github.com/photoprism/photoprism/assets/9031145/c35bd3ac-2ee8-4526-adda-9a14fa111d56)

I have gone to Library -> Scan in PhotoPrism after adding this folder to my NAS folder. However, when I check the folder in PhotoPrism, there is only 1 video in it.

![image](https://github.com/photoprism/photoprism/assets/9031145/fd5fc209-153d-4923-9532-3dbe6aaea818)

I have also added PHOTOPRISM_ORIGINALS_LIMIT = -1 into my container environment variable (it wasn't set before) and restarted my container and re-run the Scan.

![image](https://github.com/photoprism/photoprism/assets/9031145/3ac8b3df-ef1b-4635-b643-0b1274231895)

But I still get these errors:

![image](https://github.com/photoprism/photoprism/assets/9031145/76f93e8d-6f9a-4fc6-88c3-19e7bdae1020)

It says scan/indexing completed, but the videos still don't show up.

How can I ensure that PhotoPrism displays all the videos in this folder?

#### 2. How can we reproduce it?

1. Point PhotoPrism to a folder.
2. Create a sub-folder in that folder.
3. Add some >1GB videos to that folder.
4. Go to Library -> Scan in PhotoPrism and complete the scan.
5. Notice that only 1 video shows up in the folder, even though there are more.

There are 14 .mp4 and 8 .mov video files in that folder.

#### 3. What behavior do you expect?

I expect all the videos in that folder to show up in PhotoPrism.

#### 4. What could be the cause of your problem?

I am not sure. It seems to only happen to folders where there is only video files (and no images). I'm not sure if the json files being there is an issue?

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

Please include sample files or screenshots that help to reproduce your problem. You can also email files or share a download link, see <https://www.photoprism.app/contact> for details.

Visit <https://docs.photoprism.app/getting-started/troubleshooting/browsers/> to learn how to diagnose frontend issues.

**Important: If it is an import, indexing or metadata issue, we require sample files and logs from you.** Otherwise, we will not be able to process your report. If it is an import problem specifically, please always provide us with an archive of the files before you imported them so we can reproduce the behavior.

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64, ARM64, ARMv7,...

PhotoPrism® CE
Build 231128-f48ff16ef

Running on Synology DS923

(b) Database Type & Version: MariaDB, MySQL, SQLite,...

MariaDB 11.1.2+maria~ubu2204

(c) Operating System Types & Versions: Linux, Windows, Android,...

Synology DSM 7.2-64570 Update 1

(d) Browser Types & Versions: Firefox, Chrome, Safari on iPhone,...

Chrome on Windows 10

(e) Ad Blockers, Browser Plugins, and/or Firewall Software?

uBlock Origin, but same issue if it is disabled.

You can find the version/build number of the app in *Settings* by scrolling to the bottom. Note that MySQL 8 support has been discontinued, see system requirements at <https://docs.photoprism.app/getting-started/#system-requirements>.

*Always provide database and operating system details if it is a backend, import, or indexing issue. Should it be a frontend issue, at a minimum we require you to provide web browser and operating system details. When reporting a performance problem, we ask that you provide us with complete information about your environment, as there may be more than one cause.*

#### 7. On what kind of device is PhotoPrism installed?

This is especially important if you are reporting a performance, import, or indexing issue. You can skip this if you're reporting a problem you found in our public demo, or if it's a completely unrelated issue, such as incorrect page layout.

(a) Device / Processor Type: Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...

Synology DS923+

(b) Physical Memory & Swap Space in GB

4 GB DDR4 ECC SODIMM

(c) Storage Type: HDD, SSD, RAID, USB, Network Storage,...

4 x Seagate IronWolf 8TB NAS Hard Drive

(d) Anything else that might be helpful to know?

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

No.

**Comment by graciousgrey** [comment]
If you have a look at the timestamp are you sure that the errors are from the indexing you did after changing the variable? Warnings from previous index runs are not deleted. If the errors are from the last indexing can you run the [show config command](https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface) to confirm that setting -1 worked over the Web UI from Synology.

Is it possible that those files are duplicates and show up in a different folder? To check this you can search for the filenames using the filename filter under Search. 

https://docs.photoprism.app/user-guide/search/filters/#filter-reference

**Comment by srahimeen** [comment]
Yes, the files are duplicates which show up in other folders, so it might be excluding them. Is there a config flag to turn off duplicate detection?

**Comment by graciousgrey** [comment]
Thank you for checking this. At this time there is no config flag to disable duplicate detection.
